### PR TITLE
add yaml file for stateless transducer ASR model

### PR DESCRIPTION
**Pull request by hainan-xv**
# What does this PR do ?

This PR adds a yaml file for the conformer transducer model with a stateless decoder.

**Collection**: 

ASR

# Changelog 

# Usage

# Before your PR is "Ready for review"
**Pre checks**:
- [x] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [x] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: examples/asr/conf/conformer/conformer_stateless_transducer_bpe.yaml [hunk 1]
```diff
@@ -0,0 +1,259 @@
+# It contains the default values for training a Conformer-Transducer ASR model with stateless decoder, large size (~120M) with Transducer loss and sub-word encoding.
+
+# Architecture and training config:
+# Default learning parameters in this config are set for effective batch size of 2K. To train it with smaller effective
+# batch sizes, you may need to re-tune the learning parameters or use higher accumulate_grad_batches.
+# Here are the recommended configs for different variants of Conformer-Transducer, other parameters are the same as in this config file.
```
Comment (fayejf) [inline examples/asr/conf/conformer/conformer_stateless_transducer_bpe.yaml:6]: Do we need to update the table below?

File: examples/asr/conf/conformer/conformer_stateless_transducer_bpe.yaml [hunk 2]
```diff
@@ -0,0 +1,259 @@
+# It contains the default values for training a Conformer-Transducer ASR model with stateless decoder, large size (~120M) with Transducer loss and sub-word encoding.
```
Comment (fayejf) [inline examples/asr/conf/conformer/conformer_stateless_transducer_bpe.yaml:1]: Could you add this to NeMo doc?  such as [here]( https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/asr/models.html#conformer-transducer )

**Review by fayejf — CHANGES_REQUESTED** [review]

File: examples/asr/conf/conformer/conformer_stateless_transducer_bpe.yaml (hunk 2 above)
Comment (titu1994) [inline examples/asr/conf/conformer/conformer_stateless_transducer_bpe.yaml:1]: This file should be part of a subfolder - not directly inside of conformed. So conformer/stateless/...yaml

File: examples/asr/conf/conformer/conformer_stateless_transducer_bpe.yaml (hunk 1 above)
Comment (titu1994) [reply examples/asr/conf/conformer/conformer_stateless_transducer_bpe.yaml:6]: +1

**Review by titu1994 — COMMENTED** [review]
LGTM, lets fix Fei's comments

**Pull request closed without merging**